? kopete/plugins/autoreplace/autoreplaceprefs.cpp
? kopete/plugins/autoreplace/autoreplaceprefs.h
? kopete/plugins/autoreplace/Makefile
? kopete/plugins/autoreplace/Makefile.in
? kopete/plugins/netmeeting/Makefile
? kopete/plugins/netmeeting/Makefile.in
Index: kopete/plugins/perlscript/configure.in.in
===================================================================
RCS file: /home/kde/kdenonbeta/kopete/plugins/perlscript/configure.in.in,v
retrieving revision 1.1
diff -u -r1.1 configure.in.in
--- kopete/plugins/perlscript/configure.in.in	31 Jan 2003 07:01:14 -0000	1.1
+++ kopete/plugins/perlscript/configure.in.in	14 May 2003 18:58:31 -0000
@@ -9,13 +9,16 @@
     CPPFLAGS="$CPPFLAGS `perl -MExtUtils::Embed -e ccopts`"
     ac_LDFLAGS_save="$LDFLAGS"
     LDFLAGS="$LDFLAGS $all_libraries"
-    AC_TRY_LINK(
-      [#include <EXTERN.h>],
-      [#include <perl.h>],
-      [static PerlInterpreter *my_perl;],
-      [ac_cv_have_libperl="yes"],
-      [ac_cv_have_libperl="no"]
-    )
+    AC_TRY_LINK([
+#include <EXTERN.h>
+#include <perl.h>
+],
+[
+static PerlInterpreter *my_perl;
+],
+ac_cv_have_libperl="yes",
+ac_cv_have_libperl="no"
+)
     LIBS="$ac_save_libs"
     LDFLAGS="$ac_LDFLAGS_save"
   ])
Index: kopete/protocols/irc/libkirc/kirc.cpp
===================================================================
RCS file: /home/kde/kdenonbeta/kopete/protocols/irc/libkirc/kirc.cpp,v
retrieving revision 1.56
diff -u -r1.56 kirc.cpp
--- kopete/protocols/irc/libkirc/kirc.cpp	26 Apr 2003 18:27:20 -0000	1.56
+++ kopete/protocols/irc/libkirc/kirc.cpp	14 May 2003 18:58:32 -0000
@@ -16,6 +16,7 @@
 */
 
 #include "kirc.h"
+#include "config.h"
 
 #include <kdebug.h>
 #include <klocale.h>
Index: kopete/kopete/kopete.cpp
===================================================================
RCS file: /home/kde/kdenonbeta/kopete/kopete/kopete.cpp,v
retrieving revision 1.139
diff -u -r1.139 kopete.cpp
--- kopete/kopete/kopete.cpp	28 Apr 2003 11:37:28 -0000	1.139
+++ kopete/kopete/kopete.cpp	14 May 2003 20:30:17 -0000
@@ -114,7 +114,8 @@
 	{
 		// Ups! the user does not have plugins selected.
 		// TODO: show "first time" wizard and let user decide which modules to load
-		modules.append("history.desktop contactnotes.desktop");
+		modules.append("history.desktop");
+		modules.append("contactnotes.desktop");
 		// Other modules to load for the first time?
 	}
 
